### DISCUSSION TITLE
xDS configuration support to adapt the cloud native (k8s)

### DISCUSSION THREAD

**Discussion by zhangbo1882** [bfenetworks/bfe#915] [Q&A (问题和回答)] [answered]
**Is your feature request related to a problem? Please describe.**
Now in cloud native world, we need a dynamic method to use API to configure the Load balancer.  
Refer to https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by iyangsj** [comment]
目前并没有计划支持xDS接口（本身也未形成一种规范）。BFE在H2将开源控制台，支持使用方通过API修改配置。

**Comment by mileszhang2016** [comment, marked as answer]
Please refer to BFE API-Server
https://github.com/bfenetworks/api-server
